### PR TITLE
lot of fix find by correction and more

### PR DESCRIPTION
**Pull request by Blodispic**
- some of timeout removed
   -  so for disconnect during a game -> now check if not offline for set online insted of timeout
- lot of console.log() remove
- update blocked user (DM (add relation at connection and add if at sendDM for check relation))
- ban unban 15 sec -> fix by try catch in back 
- remove lot of throw in gateway ( because throw in gateway do format error pop up)
- add an i++
- update pop up chanel 
   - create chanel -> 
      - chantype reset at 0 when create a chan
      - private chabn -> invited user stay when issue with chan-name
   - setting channel ->
      - lvl up password 
- cookie move in the 3 connections insted of only 1 for fix error when reload at log page

### CODE REVIEW

**Comment by Blodispic** [comment]
Tout me parait bon, faudra juste faire des tests si quand on logout au milieu d'une game, on est online ou offline

File: front/src/components/Chat/JoinLeave.tsx [hunk 1]
```diff
@@ -128,16 +128,17 @@ export function LeaveChannel(props: { channel: IChannel }) {
...
 				swal(err_message, "error");
-				setTimeout(() => {
-					if (swal && swal.close)
-					  swal.close()
-				  }, 1000);
+				// setTimeout(() => {
+				// 	if (swal && swal.close)
+				// 	  swal.close()
+				//   }, 1000);
+				//sert a rien ?
```
Comment (se-lee) [inline front/src/components/Chat/JoinLeave.tsx:141]: yeah, i agree we don't need this
I think I put like this in JoinLeave without thinking too much lol

File: front/src/components/Chat/DirectMessage.tsx [hunk 2]
```diff
@@ -92,7 +92,8 @@ function InfoFriend(props: { user: IUser }) {
 	}
 
 	const sendFriendRequest = async () => {
-		await new Promise((resolve) => setTimeout(resolve, 1000));
+		// await new Promise((resolve) => setTimeout(resolve, 1000));
+		// Ca sert a quoi ca ? peut t'on remove ?
```
Comment (se-lee) [inline front/src/components/Chat/DirectMessage.tsx:96]: maybe (Honestly I copied this code from Profile header (Friend request button) so I'm not very sure :sweat_smile: :sweat_smile:  )
If it works fine without this line we should remove

File: front/src/components/connection/Connection.tsx [hunk 3]
```diff
@@ -42,14 +42,14 @@ export default function Connection() {
                             dispatch(setUser(data.user));
                             dispatch(change_avatar(data.user.intra_avatar))
                             dispatch(oauth());
-                            setCookie('Token', data.access_token, { path: '/' });
                             dispatch(setToken(data.access_token));
                             if (!data.user.username)
-                                navigate("./sign")
+                            navigate("./sign")
```
Comment (se-lee) [inline front/src/components/connection/Connection.tsx:47]: interesting indentation lol

**Review by se-lee — APPROVED** [review]
Thanks for the fixes 
Chat part seems Ok! :100: :+1: 

(sorry i'm not very sure about other parts :pray: I trust you guys )

File: back/src/game/pong.gateway.ts [hunk 4]
```diff
@@ -54,14 +55,15 @@ export class PongGateway implements OnGatewayDisconnect {
 
 	removeInvite(id: number)
 	{
-		const i  = 0;
+		let i  = 0;
```
Comment (emmanuel-lbs) [inline back/src/game/pong.gateway.ts:58]: ???????

**Comment by Blodispic** [comment]
String -> string ?

**Comment by Blodispic** [comment]
On va manger ?

**Review by Acusanno — APPROVED** [review]
J'ai faim

**Review by emmanuel-lbs — APPROVED** [review]

**Review by KobeLaC — APPROVED** [review]

**Pull request merged**